common/djangoapps/static_replace/__init__.py

```diff
@@ -191,6 +188,7 @@ def replace_static_url(original, prefix, quote, rest):
             else:
                 # if not, then assume it's courseware specific content and then look in the
                 # Mongo-backed database
+                from static_replace.models import AssetBaseUrlConfig, AssetExcludedExtensionsConfig
```

Comment:
I assume this is pushed down in to the logic due to a startup issue. Could we tag this as a something that could use a better refactor in the future?

Reply:
As discussed, I'll add a one-liner here and elsewhere we've pushed the imports down to explain why.

Reply:
What would a better refactor possibly look like?  Just further stripping of model modules down to their barebones, such as the removal of signal definitions from ``student.models`` in this PR?

Reply:
Something like that - splitting apart files to remove the model imports from startup. But might also require more drastic refactoring.